.lintstagedrc.json

```
@@ -5,9 +5,7 @@
   ],
   "*.{tsx,ts}": [
     "eslint --fix \"packages/manager/apps/**/*.{tsx,ts,md}\"",
-    "eslint --fix \"packages/super-components/**/*.{tsx,ts}\"",
-    "prettier --write \"packages/manager/apps/**/*.{tsx,ts}\"",
-    "prettier --write \"packages/super-components/**/*.{tsx,ts}\""
+    "prettier --write \"packages/manager/apps/**/*.{tsx,ts}\""
```

We need linting and prettier for `manager-components` also
```suggestion
    "eslint --fix \"packages/manager-components/**/*.{tsx,ts}\""
    "prettier --write \"packages/manager/apps/**/*.{tsx,ts}\"",
    "prettier --write \"packages/manager-components/**/*.{tsx,ts}\""
```

---

👍